deployments/helm/hephaestus/templates/controller/cronjob.yaml

```
@@ -0,0 +1,78 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "hephaestus.manager.name" . }}-cronjob
+spec:
+  schedule: {{ .Values.controller.ibGarbageCollector.interval | quote }}
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          annotations:
+            checksum/config: {{ include (print $.Template.BasePath "/controller/secret.yaml") . | sha256sum }}
+            {{- if .Values.controller.metrics.enabled }}
+            prometheus.io/port: "{{ .Values.controller.manager.metricsPort }}"
+            prometheus.io/scrape: "true"
+            {{- end }}
+            {{- with .Values.controller.podAnnotations }}
+            {{- toYaml . | trimSuffix "\n" | nindent 8 }}
+            {{- end }}
+            {{- with .Values.podAnnotations }}
+            {{- toYaml . | nindent 8 }}
+            {{- end }}
+          labels:
+            {{- include "hephaestus.controller.labels.matchLabels" . | nindent 12 }}
+              {{- with .Values.controller.podLabels }}
+              {{- toYaml . | trimSuffix "\n" | nindent 12 }}
+              {{- end }}
+              {{- with .Values.podLabels }}
+              {{- toYaml . | nindent 12 }}
+              {{- end }}
+        spec:
+          securityContext:
+            {{- toYaml .Values.controller.podSecurityContext | nindent 12 }}
+          volumes:
+            - name: config-vol
+              secret:
+                secretName: {{ include "hephaestus.configSecretName" . }}
+          serviceAccountName: {{ include "hephaestus.serviceAccountName" . }}
+          containers:
+            - name: image-build-gc
+              securityContext:
+                {{- toYaml .Values.controller.manager.containerSecurityContext | nindent 16 }}
+              image: {{ include "hephaestus.manager.image" . }}
+              imagePullPolicy: {{ .Values.controller.manager.image.pullPolicy }}
+              volumeMounts:
+                - name: config-vol
+                  readOnly: true
+                  mountPath: /etc/hephaestus/config.yaml
+                  subPath: config.yaml
+              command: [ "hephaestus-controller" ]
+              args:
+                - run-gc
+                - --maxIBRetention=2
+                - --config=/etc/hephaestus/config.yaml
+              {{- with .Values.controller.manager }}
+              {{- if .extraEnvVars  }}
+              env:
+                {{- with .extraEnvVars }}
+                {{- include "common.tplvalues.render" (dict "value" . "context" $) | nindent 12 }}
+                {{- end }}
+              {{ end }}
+              {{ end }}
```

I think this can be simplified
```suggestion
              {{- with .Values.controller.manager.extraEnvVars  }}
              env:
                {{- include "common.tplvalues.render" (dict "value" . "context" $) | nindent 12 }}
              {{- end }}
```